.github/workflows/update-sh.yml

```
@@ -0,0 +1,29 @@
+name: update.sh
+
+on:
+  push:
+    branches:
+    - master
+  schedule:
+  - cron:  '11 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  run_update_sh:
+    name: Run update.sh script
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Run update.sh script
+      run: ./update.sh
+    - name: Commit files
+      run: |
+        git config --local user.email "workflow@github.com"
+        git config --local user.name "GitHub Workflow"
+        git add -A
+        git commit -m "Runs update.sh" || echo "Nothing to update"
+    - name: Push changes
+      uses: ad-m/github-push-action@master
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        force: true
```

Don't think force should be required here, and IIUC could even cause loss of commits:
```suggestion
        force: false
```